src/components/NewsList.tsx

```diff
@@ -0,0 +1,146 @@
+import { Link } from "react-router-dom";
+import Carousel from "./Carousel";
+import Skeleton from "react-loading-skeleton";
+
+type Props = {
+  title: string | undefined;
+  news: Article[] | number[];
+  isLoading: boolean;
+  to: string;
+};
+export default function NewsList({ title, news, isLoading, to }: Props) {
+  return (
+    <div>
+      {!title && isLoading ? (
+        <div className="w-36">
+          <Skeleton className="h-8 w-full" />
+        </div>
+      ) : (
+        <h2 className="text-[30px] tracking-[-2px] after:block after:bg-[#C1BB25] after:h-2.5 after:w-[72px] relative after:absolute after:bottom-2 font-extrabold mb-2">
+          <span className="relative z-10 text-[#454541]">{title}</span>{" "}
+        </h2>
+      )}
+
+      <div className="w-full flex gap-9 h-full">
+        <div className="flex-1 h-full">
+          {isLoading ? (
+            <Skeleton className="w-full h-[453px]" />
+          ) : (
+            <Carousel to={to} articles={news?.slice(0, 5) as Article[]} />
+          )}
+
+          <div className="mt-10 hidden  lg:flex flex-col gap-4">
+            {news?.slice(8, 12).map((article) => {
+              if (isLoading || typeof article === "number") {
```

Comment:
Do you really need to check for the type of the `article` here to render the loading state? Is the `isLoading` state not enough by itself?